pkg/repository/v1manifest/version.go

```diff
@@ -0,0 +1,10 @@
+package v1manifest
+
+import (
+	"github.com/pingcap-incubator/tiup/pkg/repository/v0manifest"
+)
+
+type (
+	// Version represents a version string, like: v3.1.2
+	Version = v0manifest.Version
```

Comment:
```suggestion
	Version string
```

I like to keep the v0 and v1 manifest systems separate. However, I think it is also ok to just use a `string` for the version rather than create a type because we don't ensure any further invariants when we use the `Version` type.

Reply:
This file is deleted now.